handbook/people.md

```diff
@@ -46,8 +46,35 @@ For more developed thoughts about __spending guidelines and limits__, please rea
 ## Meetings
 
 * At Fleet, meetings start whether you're there or not. Nevertheless, being even a few minutes late can make a big difference and slow your meeting counterparts down. When in doubt, show up a couple of minutes early.
-* It's okay to spend the first minute or two of a meeting to be present and make small talk, if you want.  Being all-remote, it's easy to miss out on hallway chatter and human connections that happen in [meatspace](https://www.dictionary.com/browse/meatspace).  Why not use this time together during the first minute to say "hi".  Then you can jump right in to the topics being discussed?
+* It's okay to spend the first minute or two of a meeting to be present and make small talk, if you want.  Being all-remote, it's easy to miss out on hallway chatter and human connections that happen in [meatspace](https://www.dictionary.com/browse/meatspace).  Why not use this time together during the first minute to say "hi"?  Then you can jump in to the topics being discussed.
 * Turning on your camera allows for more complete and intuitive verbal and non-verbal communication.  When joining meetings with new participants who you might not be familiar with yet, feel free to leave your camera on or to turn it off.  When you lead or cohost a meeting, turn your camera on.
+* Consider: Does this need to be a meeting? Are all participants required? Being all-remote, it can
+  be useful to get some "face time", but this must be balanced with focused work.
```

Comment:
How do you feel about:

> - In an all-remote company, “face time” matters.  Just remember: even if someone’s calendar is open, they have other work to do.  Limiting (or batching up) internal meetings can enable longer, uninterrupted stretches of deep work.

Reply:
Yes I like this! Next time can you please use the `+/-` button to submit it as a suggestion so I can just auto-apply it?

Reply:
Will do!  (I looked but couldn't find it on mobile web)